dirac.cfg

```
@@ -582,6 +582,11 @@ Systems
         index_name=a_different_name
       }
     }
+    JobWrapper
+    {
+      # Minimum output buffer requested for running jobs
+      MinOutputDataBufferGBs = 3
```

I guess we want to keep the same default?
```suggestion
      MinOutputDataBufferGB = 5
```

---

I allowed myself to make this singular per your other comment.